packages/gatsby-theme-newrelic/src/components/GlobalNavLink.js

```diff
@@ -35,21 +35,22 @@ const GlobalNavLink = ({ children, href, activeSite }) => {
         display: flex;
         align-items: center;
         height: 100%;
-        padding: 0.25rem 9px;
-        color: var(--secondary-text-color);
-        font-size: 0.6875rem;
+        padding: 4px 9px;
```

Comment:
```suggestion
        padding: 0.25rem 0.625rem;
```

Huh, not sure why we mixed units here. Could we use rem for padding instead?